en/guide/overview.md

```diff
@@ -1,13 +1,56 @@
-* Advertises and shares RTSP video streams for all connected cameras. These can be consumed by the GCS and other video player services.
-* Captures and stores/shares video and still images for all connected cameras via the [MAVLink Camera Protocol](https://mavlink.io/en/protocol/camera.html). GCS requests for camera actions are forwarded by PX4. 
 
-> **Note** The *Camera Streaming Daemon* also integrates with Gazebo, providing a simulated camera backend. In this case the diagram/configuration can be slightly different as the GCS can communicate directly with the server (there is no need for messages to be forwarded by PX4).
\ No newline at end of file
+## Key Features
+
+CSD supports the following key features:
+
+* Automatically attaches [compatible cameras](#supported_cameras) connected to the Linux computer.
+* RTSP video streaming from *all* connected cameras (for consumption by GCS or other video players).
+* RTSP video stream advertising/discovery using Avahi.
+* [MAVLink Camera Protocol](#mavlink_support) support for up to 5 cameras, enabling video and image capture and storage, and querying/setting camera options.
+* Gazebo simulated camera backend (so you can view video streams from within a simulated environment)!
+* Configurable back-end that can be extended to interface with new types of cameras and new front-end protocols.
+
+
+## Supported Cameras {#supported_cameras}
+
+Cameras that support the [Video4Linux (V4L2) API](https://linuxtv.org/downloads/v4l-dvb-apis/uapi/v4l/v4l2.html) should work out of the box. 
+The server can be [extended](../guide/extending_csd.md) to support other back-end camera protocols/APIs.
+
+Some readily-available cameras that have been used in testing include: 
+* Logitech C270 HD Webcam
+* Sony PlayStation Eye Camera
+* Intel RealSense 3D Camera
+
+## Configuration Options {#configuration}
+
+Compile-time configuration is provided through [build-configuration flags](../getting_started/building_installation.md#configure). These allow developers to enable support for: MAVLink, RTSP stream discovery using Avahi, RealSense Camera, Intel Aero bottom camera and Gazebo. RTSP streaming is always enabled.
+
+Run-time configuration is provided by the [CSD Configuration File](../guide/configuration_file.md). Among other things, this allows users to set where captured images will be stored, devices that will not be connected, and settings required for defining the MAVLink node.
+
+Advanced configuration information about individual cameras is specified in [Camera Definition Files](TBD). While not strictly part of CSD, these files are referenced in the [CSD Configuration File](../guide/configuration_file.md#uri) and may need to be served on the same computer as CSD.
+
+
+## MAVLink Camera Protocol Implementation {#mavlink_support}

```

Comment:
Please check this in particular - is it accurate/anything else to say about what we support w.r.t. the protocol?

Reply:
video capture is not supported yet